litmus-portal/frontend/src/theme/index.tsx

```diff
@@ -136,6 +150,13 @@ function customTheme(options: ThemeOptions) {
         passedTests: 'rgba(16, 155, 103, 0.2)',
         failedTests: 'rgba(202, 44, 44, 0.2)',
       },
+      projectDisplayColor: 'rgba(128, 128, 128)',
+      buttonEditProfile: 'rgba(0, 0, 0)',
+      homePageCardBackgroundColor: '#FFFFFF',
+      homePageWorkflowCardBackgroundColor: '#FFFFFF',
+      workflowTitleColor: '#5B44BA',
```

Comment:
this one is same as secondary.dark...in the other projects do we not have this as secondary.dark? If not we can define it like this

Reply:
Yes, these are the exceptional case, in which we are not using the secondary color in other design